### PR TITLE
PLF-8192: Applications in spaces will be accessible by right click

### PR DESCRIPTION
**Pull request by AzmiTouil**
Change href attribute and delete the onclick method.

### CODE REVIEW

File: extension/war/src/main/webapp/groovy/social/webui/space/UISpaceMenu.gtmpl [hunk 1]
```diff
@@ -111,24 +111,21 @@ if(appSelected == null || appSelected.length() == 0) {
...
-String spaceHomeAppLabel = _ctx.appRes(uicomponent.getId() + ".label.SpaceHomePage");
-%>
 
 <ul class="nav nav-tabs spaceMenuTab" id="spaceMenuTab" style="visibility: hidden;">
 <li class="$selectedClazz">
-<a data-toggle="tab" <%=activityStreamUrl%>>
+<a <%=activityStreamUrl%>>
         <i class="uiIconAppSpaceHomePage uiIconDefaultApp"></i>
-<span class="tabName"> $spaceHomeAppLabel</span>
+<span class="tabName">Activity Stream</span>
```
Comment (plamarque) [inline extension/war/src/main/webapp/groovy/social/webui/space/UISpaceMenu.gtmpl]: Must be localized

Comment (AzmiTouil) [reply extension/war/src/main/webapp/groovy/social/webui/space/UISpaceMenu.gtmpl]: Okay

Comment (AzmiTouil) [reply extension/war/src/main/webapp/groovy/social/webui/space/UISpaceMenu.gtmpl]: done

File: extension/war/src/main/webapp/groovy/social/webui/space/UISpaceMenu.gtmpl [hunk 2]
```diff
@@ -126,7 +126,7 @@ String spaceHomeAppLabel = _ctx.appRes(uicomponent.getId() + ".label.SpaceHomePa
 
 <ul class="nav nav-tabs spaceMenuTab" id="spaceMenuTab" style="visibility: hidden;">
 <li class="$selectedClazz">
-<a data-toggle="tab" <%=activityStreamUrl%>>
```
Comment (thomasdelhomenie) [inline extension/war/src/main/webapp/groovy/social/webui/space/UISpaceMenu.gtmpl]: Why do we remove `data-toggle="tab"` ?

Comment (AzmiTouil) [reply extension/war/src/main/webapp/groovy/social/webui/space/UISpaceMenu.gtmpl]: Using a href for a tab is not allowed in bootstrap, I have tested the href with data-toggle="tab" and didn't work.
PS : https://stackoverflow.com/questions/29567846/href-doesnt-work-when-using-data-toggle-tab?answertab
WDYT ?

Comment (thomasdelhomenie) [reply extension/war/src/main/webapp/groovy/social/webui/space/UISpaceMenu.gtmpl]: ok, data-toggle="tab" is something from bootstrap, I was not aware of that.
I agree it should be removed then.

**Pull request merged**